index.html

```diff
@@ -3417,16 +3414,18 @@ <h3>Refreshing</h3>
     }
   },
   <span class="highlight">"refreshService": {
-    "id": "https://university.example/refresh/3732",
-    "type": "ManualRefreshService2018"
+    "type": "VerifiableCredentialRefreshService2021",
```

Comment:
As it stands, the type will expand to `https://www.w3.org/ns/credentials/examples#VerifiableCredentialRefreshService2021` because the term itself is not defined in the core context file. Is that the intention? Or should we add a CCG-maintained context file somewhere (maybe as a local context)?

Reply:
Yes, good point, I'll add the refresh service context in an update to this PR.

Reply:
Fixed in 1dff5c219f2408f92aa0d406f47a6ce233b5d16b.